pkg/kev/config/schema.go

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2020 Appvia Ltd <info@appvia.io>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+var ServicesSchema = map[string]interface{}{
+	"$schema": "http://json-schema.org/draft/2019-09/schema#",
+	"$id":     "http://appvia.io/schemas/kev-service-labels-schema.json",
+	"type":    "object",
+	"properties": map[string]interface{}{
+		LabelComponentEnabled:                  map[string]interface{}{"type": "string", "enum": []string{"false", "true"}},
+		LabelServiceType:                       map[string]interface{}{"type": "string", "enum": []string{"None", "Headless", "ClusterIP", "Nodeport", "LoadBalancer"}},
```

Comment:
Is validator going to be case insensitive? We should support mixed cases e.g. `headless` == `HeAdLESs` etc..

Reply:
Nope, for now, it is case sensitive. But it will flag the expected values on error, e.g. `error ... expected:  "None", "Headless", "ClusterIP", "Nodeport", etc..`.

On a first look, supporting mixed case with JSON schema seems to be a bit of a hassle,
- **Proposal from 2013**: https://groups.google.com/g/json-schema/c/V-ZAJfV7mt8
- **Ugly**: https://stackoverflow.com/questions/60001238/how-to-make-the-json-schema-flexible-enough-to-handle-case-sensitive-input-for-e

Maybe leave that for another ticket and see if users find it annoying?

Reply:
Sure, can leave as is for now. Maybe check the docs if it matches.